src/stacks-hierarchy/Root_PurchaseOverviewScreen/use-offer-state.ts

```diff
@@ -142,6 +143,11 @@ export function useOfferState(
 
   const updateOffer = useCallback(
     async function (cancelToken?: CancelToken) {
+      if (skip) {
```

Comment:
This is basically is to skip request for prices when we know that is already free. 
A better solution? :) is welcome.

Reply:
I think it is ok. Many other hooks have a flag named `disabled`, would that also be appropriate naming here?